l10n_nl_bsn/models/res_partner.py

```diff
@@ -66,3 +65,30 @@ def _warn_bsn_existing(self):
             'message': msg % (self.name, self.bsn_number)
         }
         return warning
+
+    @api.model
+    def search(self, args, offset=0, limit=None, order=None, count=False):
+        res_domain = []
+        for domain in args:
+            if (
+                    len(domain) > 2 and domain[0] == 'bsn_number' and
+                    isinstance(domain[2], str) and domain[2]
+            ):
+                operator = domain[1]
```

Comment:
I'd suggest to do this only if `operator` is not in https://github.com/OCA/OCB/blob/12.0/odoo/osv/expression.py#L151 - otherwise we get false positives I think.

I also think it should be possible to turn off this functionality via some context key.

Reply:
Thanks. Done